--- a/pppoe/nss_connmgr_pppoe.c	2020-07-25 21:17:58.465972099 +0200
+++ b/pppoe/nss_connmgr_pppoe.c	2020-07-25 21:21:13.734713515 +0200
@@ -1,6 +1,6 @@
 /*
  **************************************************************************
- * Copyright (c) 2017-2019, The Linux Foundation. All rights reserved.
+ * Copyright (c) 2017-2020, The Linux Foundation. All rights reserved.
  * Permission to use, copy, modify, and/or distribute this software for
  * any purpose with or without fee is hereby granted, provided that the
  * above copyright notice and this permission notice appear in all copies.
@@ -28,6 +28,7 @@
 #include <linux/if_pppox.h>
 #include <net/ip.h>
 #include <linux/if_bridge.h>
+#include <net/bonding.h>
 #ifdef CONFIG_OF
 #include <linux/of.h>
 #endif
